example/src/ParallaxDrawer.tsx

```diff
@@ -63,30 +63,42 @@ const MyNavScreen = ({
           }}
           placeholder="Focus this TextInput then drag the drawer!"
         />
-        <Button onPress={() => navigation.openDrawer()} title="Open drawer" />
+        <Button 
+          onPress={() => {
+            navigation.openDrawer(); 
+            navigation.emit('drawerOpen');
```

Comment:
We shouldn't need to emit these events here in user code. Opening/closing drawer should always emit these events internally